xpublish/rest.py

```diff
@@ -104,55 +104,59 @@ def zmetadata_json(self):
         )
         return Response(echunk, media_type="application/octet-stream")
 
-    @property
-    def app(self):
-        """ FastAPI app """
-        if self._app is None:
+    def init_app(self, **kwargs):
```

Comment:
Can you add a docstring for this method?

Also, can you point me to the API docs for the FastAPI constructor? I'm having trouble locating them on the web docs.

Reply:
Along these lines, do you think its worth adding a little blurb on how to customize the deployment to the docs? Something [here](https://xpublish.readthedocs.io/en/latest/tutorial.html)?